features/employers/benefit_applications/create_off_cycle_benefit_application.feature

```diff
@@ -0,0 +1,126 @@
+Feature: Ability for employer to create an off-cycle benefit application and allow employees to plan shop
+
+  Background: Setup site, employer, and benefit market catalogs
+    Given the shop market configuration is enabled
+    Given all announcements are enabled for user to select
+    Given a CCA site exists with a benefit market
+    And benefit market catalog exists for enrollment_open renewal employer with health benefits
+    And there is an employer ABC Widgets
+    And ABC Widgets employer has a staff role
+
+  Scenario: Initial Employer is terminated. Employer is able to create off-cycle benefit application and employees are able to plan shop
```

Comment:
these scenarios seem excessively long - can we split them up or think about other ways to test this?